### ISSUE TITLE
My accuracy of SVTP of lexicon0 is 80.9 rather than the ReadMe's 81.2,  is there somewrong?

### ISSUE THREAD

**Issue by sonack** [ayumiymk/aster.pytorch#22]
![image](https://user-images.githubusercontent.com/12935189/66270490-e8c50b00-e886-11e9-8ee8-5c505a9335b7.png)
Is there something wrong?

**Comment by sonack** [comment]
FYI, Below is my tested your pretrained models' performace:

|               | IIIT5k |  SVT |  IC03 |  IC13 |  IC15 | SVTP  |  CUTE |
|:-------------:|:------:|:----:|:-----:|:-----:|:-----:|:-----:|:-----:|
|  ASTER (L2R)  |  92.67 |   -  | 93.72 | 90.74 |    -  | 78.76 | 76.39 |
| ASTER.Pytorch |  93.2  | 89.2 | 92.2  |   91  |  78.0 |  **81.2** |  81.9 |
| ASTER.Pretrain|  93.2  | 89.2 | **92.8**  | **93.3**  |  78.0 |  80.9 |  81.9 |

**Comment by ayumiymk** [comment]
emmm... Except for SVTP, you can obtain the same even better results on other datasets. Maybe the images of SVTP I used is a little different from you. But I think 0.3 should be ok. If I have time, I will check that.

**Comment by sonack** [comment]
> emmm... Except for SVTP, you can obtain the same even better results on other datasets. Maybe the images of SVTP I used is a little different from you. But I think 0.3 should be ok. If I have time, I will check that.

I used the tensorflow version of Aster's released test datasets https://github.com/bgshih/aster/releases/, so may it be a little different from yours?

**Comment by ayumiymk** [comment]
I can guarantee that the original images in both the tensorflow and pytorch versions are the same. However, there may be some different image processings to generate tfrecoder and lmdb. You can check the `create_xxx.py` to find the reasons.

**Comment by sonack** [comment]
> I can guarantee that the original images in both the tensorflow and pytorch versions are the same. However, there may be some different image processings to generate tfrecoder and lmdb. You can check the `create_xxx.py` to find the reasons.

thanks, I just replace the proper file to be gt.txt and use your `https://github.com/ayumiymk/aster.pytorch/blob/master/lib/tools/create_svtp_lmdb.py` file.

**Comment by shaohailin** [comment]
@sonack  icdar2013 in the paper has 1015 images,but the dataset in https://github.com/bgshih/aster/releases/ only has 857 images 
i use the 857 images of icdar2013 and test it ,the result is 93.3%
@ayumiymk  could you sent the link of the icdar2013_1015 dataset and it's gt.txt in the paper ,
